website/package.json

```diff
@@ -22,14 +25,15 @@
     "autoprefixer": "^10.4.14",
     "clsx": "^1.2.1",
     "postcss": "^8.4.21",
-    "prism-react-renderer": "^1.3.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-live": "^4.0.0",
+    "prism-react-renderer": "*",
```

Comment:
should this be an explicit version, since PRR isn't part of this workspace?

Reply:
It isn't, but the `*` also means, I believe, use the version specified in the root package.json too.